synapse/cores/common.py

```diff
@@ -574,60 +549,51 @@ def _cortex_condefs():
         confdefs = (
 
             ('dir', {'type': 'str',
-                        'doc': 'The cortex metadata directory'}),
+                'doc': 'The cortex metadata directory'}),
 
-            ('fifo:defs', {'defval': {}, 'doc': 'Config defaults for core fifos'}),
+            ('fifo:defs', {'defval': {},
```

Comment:
I would just delete this as it seems to be unused at this point.

Reply:
fixed